clustering/web/undertow/src/main/java/org/wildfly/clustering/web/undertow/session/DistributableSessionManager.java

```diff
@@ -150,6 +152,13 @@ public void accept(HttpServerExchange exchange) {
         }
         // Workaround for UNDERTOW-1902
         if (exchange.isResponseStarted()) { // Should match the condition in io.undertow.servlet.spec.HttpServletResponseImpl#isCommitted()
+            // The servlet specification mandates that an ISE be thrown here
```

Comment:
@pferraro If I get it correctly, this workaround will go after UNDERTOW-1902 is merged and released, is that right?

Reply:
After UNDERTOW-1902, this method will not be invoked unless DeploymentInfo.isOprhanSessionAllowed() returns true (so we do not need to worry about throwing an ISE), however, the distributed session manager still needs to return an OrphanSession in this case, since the session will never be scheduled for expiration.